client/state/sites/plans/selectors.js

```diff
@@ -126,6 +122,24 @@ export function getPlanRawDiscount(
 		: plan.rawDiscount;
 }
 
+/**
+ * Returns if a plan is discounted
+ *
+ * @param  {Object}  state         global state
+ * @param  {Number}  siteId        the site id
+ * @param  {String}  productSlug   the plan product slug
+ * @return {Boolean}               true if a plan has a discount
+ */
+export function isPlanDiscounted(
+	state,
+	siteId,
+	productSlug
+) {
+	const plan = getSitePlan( state, siteId, productSlug );
```

Comment:
Shouldn't we return `null` if there is no plan? I think this has been a convention for selectors.